### DISCUSSION TITLE
Double cut

### DISCUSSION THREAD

**Discussion by CBYTLSE** [arkypita/LaserGRBL#2165] [Q&A]
Hi,
I use Inscape to draw a disk including its external circle. I export a png of all the elements in order to engrave them. 
No problem for the engraving.

By GRBL in a second time I use only the external circle in order to cut the disk  by an append in GRBL.
The external circle is a png format, during the cut with GRBL I got a double cut (2*360°). I got the disk with no problem (2 cuts)!!!

What format  must I use for the append (png…) and what option (vectorisation, Central line Vectorisation…) ?
Vectorisation creates 2 small non cutted sections at 90°
Does the line width  of the circle has to be taken into account ?

Regards

**Comment by Tribo-2dot0** [comment]
Exporte o círculo de corte como svg.
